app/views.py

```diff
@@ -284,8 +284,7 @@ def finalize_or_reject(request, process_id):
             background = Background(process, item_type, item_id, role, user)
             background.processing()
             if user_type == "public":
```

Comment:
Why do we have this change?

Reply:
I didn't make that change. I made a pull request maybe it didn't update completely before I make the changes. I will make a pull request again and see